src/modules/compliance/src/lib/procedures/ufwStatus.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

File names should be PascalCased. Can we do 'UfwStatus.cpp", etc?

Reply:
We will fix this for all files in a separate PR.